src/api/models.py

```diff
@@ -242,534 +241,846 @@ class ShotModel(SQLModel, table=True):
+    cpf_p05011: Optional[float] = Field(alias="cpf_s_sfps_mcs")
 
-    cpf_p05015: Optional[float] = Field(nullable=True)
+    cpf_p05015: Optional[float] = Field(alias="cpf_s_scs4_mcs")
 
-    cpf_p05016: Optional[float] = Field(nullable=True)
+    cpf_p05016: Optional[float] = Field(alias="cpf_s_fa3_mcs")
 
-    cpf_p05017: Optional[float] = Field(nullable=True)
+    cpf_p05017: Optional[float] = Field(alias="cpf_s_fa4_mcs")
 
-    cpf_p05025: Optional[float] = Field(nullable=True)
+    cpf_p05025: Optional[float] = Field(alias="cpf_s_p1ps_mcs")
 
-    cpf_p05027: Optional[float] = Field(nullable=True)
+    cpf_p05027: Optional[float] = Field(alias="cpf_s_cp3c_mcs",
+                                        description="(MCS Setup) select P3 counterpulse bank for MAST only")
 
-    cpf_p05028: Optional[float] = Field(nullable=True)
+    cpf_p05028: Optional[float] = Field(alias="cpf_s_cp4_mcs",
+                                        description="(MCS Setup) select P4 start bank for MAST only")
 
-    cpf_p05029: Optional[float] = Field(nullable=True)
+    cpf_p05029: Optional[float] = Field(alias="cpf_s_cp5_mcs",
+                                        description="(MCS Setup) select P3 start bank for MAST only")
 
-    cpf_p05030: Optional[float] = Field(nullable=True)
+    cpf_p05030: Optional[float] = Field(alias="cpf_s_p3_ps_mcs",
+                                        description="(MCS Setup) select Siemens & GEC switches for P3 plasma Initiation for MAST only")
 
-    cpf_p05032: Optional[float] = Field(nullable=True)
+    cpf_p05032: Optional[float] = Field(alias="cpf_s_p4_thy_mcs",
+                                        description="(MCS Setup) select P4 Ignitron and Thyristor circuit for MAST only")
 
-    cpf_p05033: Optional[float] = Field(nullable=True)
+    cpf_p05033: Optional[float] = Field(alias="cpf_s_p5_thy_mcs",
+                                        description="(MCS Setup) select P5 Ignitron and Thyristor circuit for MAST only")
 
-    cpf_p05153: Optional[float] = Field(nullable=True)
+    cpf_p05153: Optional[float] = Field(alias="cpf_s_ecrh_mcs")
 
-    cpf_p06000: Optional[float] = Field(nullable=True)
+    cpf_p06000: Optional[float] = Field(alias="cpf_s_xmm_os9_mcs")
 
-    cpf_p06001: Optional[float] = Field(nullable=True)
+    cpf_p06001: Optional[float] = Field(alias="cpf_s_xpc_os9_mcs")
 
-    cpf_p06002: Optional[float] = Field(nullable=True)
+    cpf_p06002: Optional[float] = Field(alias="cpf_s_xcm_os9_mcs")
 
-    cpf_p06003: Optional[float] = Field(nullable=True)
+    cpf_p06003: Optional[float] = Field(alias="cpf_s_xmw_os9_mcs")
 
-    cpf_p06004: Optional[float] = Field(nullable=True)
+    cpf_p06004: Optional[float] = Field(alias="cpf_s_xma_os9_mcs")
 
-    cpf_p10963: Optional[float] = Field(nullable=True)
+    cpf_p10963: Optional[float] = Field(alias="cpf_ss_nbi_gas_mcs",
+                                        description="(MCS Setup) MCS_SS_NBI_gas_species for MAST only")
```

Comment:
Good to add descriptions to each of the fields, but we can go further.

Can we change the naming conventions?

For example, remove the `cpf_` and I would put a group short hand at the start (so that it works better with auto complete etc.). So:

```
mcs_ss_nbi_gas
```

I think we should do the same for other groups, so:

```
nbi_*
plasma_current_*
equilibrium_*
```
etc.